tests/plugins/easyimage/uploadintegrations.js

```diff
@@ -126,6 +127,11 @@
 			},
 
 			setUp: function() {
+				// We need to ignore entire test suit to prevent of fireing init, which breaks test suit on IE8-IE10.
+				if ( !this.editor.plugins.easyimage.isSupportedEnvironment() ) {
```

Comment:
Not sure if we should join this `if` with the below one?  To me, it is cleaner when they are separated. WDYT?

Reply:
I also think the same – that's why I left them separated.